R/class-comment.R

```diff
@@ -118,7 +118,7 @@ wb_comment <- function(text = NULL,
                        width = 2,
                        height = 4) {
   # Code copied from the wbWorkbook
-  author <- author %||% Sys.getenv("USERNAME")
+  author <- author %||% Sys.getenv("USERNAME") %||% Sys.getenv("USER")
```

Comment:
Oops, I just remember that `Sys.getenv()` returns "" when it can't find something. This will not work as intended.

Reply:
Can we grep for USER in the environment? Is it an issue with Windows? I tested with Linux and Mac and afaik they all had USERNAME

Reply:
For me (Windows) `Sys.getenv("USER")` returns `""` and `Sys.getenv("USERNAME")` works as expected. Let's drop "USER" then?

Reply:
👍 you introduced it, you can remove it

Reply:
Darn! I lied. Sorry for the confusion :-(
My Mac shows
```R
> Sys.getenv("USERNAME")
[1] ""
> Sys.getenv("USER")
[1] "janmarvingarbuszus"
```

maybe something like:

```R
ifelse(Sys.getenv("USERNAME") == "", Sys.getenv("USER"), Sys.getenv("USERNAME"))
```

Reply:
Thanks for validating. I will reuse the `unset` argument of `Sys.getenv()` instead of ifelse.